static/js/publisher/market/stickyListingBar.js

```diff
@@ -0,0 +1,20 @@
+export default function() {
+  const stickyBar = document.querySelector(".snapcraft-p-sticky");
+  const listings = document.querySelector(
+    ".u-float--left.p-heading--three.u-no-margin--bottom"
+  );
+
+  const observer = new IntersectionObserver(entries => {
```

Comment:
`IntersectionObserver` has quite limited browser support. It's not supported by IE, Safari possibly others. This means it will fail with an error in these browsers.

Reply:
Just added an `eventListener` on scroll for cross-browser support